web/react/components/post_info.jsx

```diff
@@ -30,9 +30,7 @@ module.exports = React.createClass({
                         <div className="dropdown">
                         { isOwner || (this.props.allowReply === "true" && type != "Comment") ?
                             <div>
-                                <a href="#" className="dropdown-toggle theme" type="button" data-toggle="dropdown" aria-expanded="false">
-                                    [...]
-                                </a>
+                                <a href="#" className="dropdown-toggle theme" type="button" data-toggle="dropdown" aria-expanded="false" />
```

Comment:
Is there any issue with the self-closing anchor tags? I don't see any issue with them when I use this in Chrome, but I'm not sure if you're supposed to be able to have those. It might not matter though since this is in the JSX.